include/geometry/triangle.h

```diff
@@ -0,0 +1,22 @@
+#ifndef GEOMETRY_TRIANGLE_H
+#define GEOMETRY_TRIANGLE_H
+
+#include <geometry/point.h>
+#include <geometry/vetor.h>
+
+struct Triangle {
+    Point vertices[3]; 
+
+    Triangle();
+    Triangle(const Point& v1, const Point& v2, const Point& v3);
+
+    double area() const;  // area of the triangle
+    Point centroid() const;  // centroid of the triangle
+    Vetor normal() const;  // normal vector of the triangle
+    bool contains(const Point& p) const;  // check if the triangle contains a point
```

Comment:
```suggestion
    double area() const;
    Point centroid() const;
    Vetor normal() const;
    bool contains(const Point& p) const; 
```

Remove esses comentários e adiciona um método chamado `getTriangle`, sugestão de @kinhosz.